four.html

```diff
@@ -127,15 +127,23 @@ <h2 id="logos"><a href="#logos">Logos</a></h2>
       </main>
     </div>
 
+
     <div class="row section">
       <aside class="col">
-        <h2 id="float-and-clear"><a href="float-and-clear">Float and Clear</a></h2>
+        <h2 id="float-and-clear"><a href="#float-and-clear">Float and Clear</a></h2>
       </aside>
       <main class="col-9">
         <ul class="points">
           <li><a href="https://www.w3schools.com/css/css_float.asp" target="_blank">Float Property</a></li>
           <li><a href="https://css-tricks.com/almanac/properties/c/clear/" target="_blank">Clear Property</a></li>
-          <li><a href="https://www.youtube.com/watch?v=xara4Z1b18I&t=8s">Use them together (Video)</a></li>
+          <li><a href="https://www.youtube.com/watch?v=xara4Z1b18I&t=8s" target="_blank">Use them together (Video)</a></li>
+          <li><a href="https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Block_formatting_context" target="_blank">Block Formatting Context</a></li>
+          <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flow_Layout/Intro_to_formatting_contexts#bfc_creation_examples" target="_blank">Problems with Floats</a>            <ul>
+              <li>where meant for image positioning</li>
```

Comment:
I think there's a typo here and the next line: "where" might need to be edited to "were"?

Reply:
Good catch! Fixed!